LICENSE

```diff
@@ -88,75 +88,11 @@ in this Software without prior written authorization of the copyright holder.
 ******************************************************************************
 * libwww license
 *
-* CiscoEST uses HTTP header parsing logic from libwww
+* libEST uses HTTP header parsing logic from libwww
 ******************************************************************************
 libwww: W3C's implementation of HTTP can be found at: http://www.w3.org/Library/
 
 Copyright © 1994-2000 World Wide Web Consortium, (Massachusetts Institute of Technology, Institut National de Recherche en Informatique et en Automatique, Keio University). All Rights Reserved. This program is distributed under the W3C's Software Intellectual Property License. This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See W3C License http://www.w3.org/Consortium/Legal/ for more details. 
 
 Copyright © 1995 CERN. This product includes computer software created and made available by CERN. This acknowledgment shall be mentioned in full in any product which includes the CERN computer software included herein or parts thereof.
 ******************************************************************************
-
-
```

Comment:
We need to adjust this.  Turns out this not only needs to be kept in the libEST version, it also needs to be added to the CiscoEST version.